pkg/apiserver/logsearch/models.go

```diff
@@ -97,6 +97,7 @@ type TaskModel struct {
 	Target       *utils.RequestTargetNode `json:"target" gorm:"embedded;embedded_prefix:target_"`
 	State        TaskState                `json:"state" gorm:"index"`
 	LogStorePath *string                  `json:"log_store_path" gorm:"type:text"`
+	Size         int64                    `json:"size" gorm:"index"`
```

Comment:
maybe uint64 is better?

Reply:
os.Stat return `int64`, so I think it is not necessary to use `uint64`. 🤔